src/test-utils/dom/code-view/index.ts

```diff
@@ -6,8 +6,8 @@ import styles from "../../../code-view/styles.selectors.js";
 export default class CodeViewWrapper extends ComponentWrapper {
   static rootSelector: string = styles.root;
 
-  findContent(): ElementWrapper {
-    return this.findByClassName(styles.code)!;
+  findContent() {
```

Comment:
changing the signature of this is not a backwards compatible change

Reply:
Updated this to be backwards compatible.